### PR TITLE
Fix to load winer NER data

### PR DESCRIPTION
**Pull request by lizgzil**
Fixes https://github.com/wellcometrust/WellcomeML/issues/91

Small fix that I noticed when I tried to use the data.

It didn't come up in the test since I the test was for the first data point (which this bug didn't effect), so adding a test for a later data point instead.

### CODE REVIEW

**Comment by lizgzil** [comment]
🤦 test_sent2vec.py failed

**Comment by aCampello** [comment]
> 🤦 test_sent2vec.py failed

Should we organise a war cabinet to fix that - it's blocking my other PR too :(

**Comment by nsorros** [comment]
> > 🤦 test_sent2vec.py failed
> 
> Should we organise a war cabinet to fix that - it's blocking my other PR too :(



> > 🤦 test_sent2vec.py failed
> 
> Should we organise a war cabinet to fix that - it's blocking my other PR too :(

happy to do that, bad sent2vec

File: wellcomeml/datasets/winer.py [hunk 1]
```diff
@@ -216,8 +216,8 @@ def _load_data_spacy(data_path, inc_outside=True, merge_entities=True):
     X = []
     Y = []
     sentence_text = None
+    char_i = 0 # A counter to populate the start and end character indexes for each entity
```
Comment (nsorros) [inline wellcomeml/datasets/winer.py:219]: char_i is not a good name, hence the comment. ent_start_index could be a better one

**Comment by aCampello** [comment]
If you rebase to master the tests should pass (just merged https://github.com/wellcometrust/WellcomeML/pull/92 that skips the problematic test)

**Pull request merged**